apps/web/.env.sample

```
@@ -19,3 +17,14 @@ VITE_POSTHOG_API_KEY=""
 VITE_POSTHOG_HOST=""
 
 LOOPS_KEY=""
+
+SUPABASE_ANON_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)